beta-20161013.md

```diff
@@ -0,0 +1,121 @@
+- The [`ALTER TABLE ... SPLIT AT`](alter-table.html) command can now be used to force a range split at a specified key. [#8938](https://github.com/cockroachdb/cockroach/pull/8938)
+- Added new `information_schema` metatables as well as initial support for the `pg_catalog` database. This work is part of our ongoing effort to provided standard database introspection required by popular ORMs. [#8433](https://github.com/cockroachdb/cockroach/pull/8433), [#8498](https://github.com/cockroachdb/cockroach/pull/8498), [#9565](https://github.com/cockroachdb/cockroach/pull/9565), [#9114](https://github.com/cockroachdb/cockroach/pull/9114), [#9104](https://github.com/cockroachdb/cockroach/pull/9104)
+- [`CHECK`](constraints.html#check) constraints can now be added with the [`ALTER TABLE ADD CHECK`](alter-table.html) and `ALTER TABLE VALIDATE` statements. [#9127](https://github.com/cockroachdb/cockroach/pull/9127) [#9152](https://github.com/cockroachdb/cockroach/pull/9152)
+- The [`ALTER TABLE DROP CONSTRAINT`](alter-table.html) statement can now drop [`CHECK`](constraints.html#check) and [`FOREIGN KEY`](constraints.html#foreign-key) constraints. [#8747](https://github.com/cockroachdb/cockroach/pull/8747)
+- The [`SHOW CONSTRAINTS`](show-constraints.html) statement now requires the user to have privileges for the requested table. [#8658](https://github.com/cockroachdb/cockroach/pull/8658)
+- The [`EXPLAIN`](explain.html) statement now works correctly for `VALUES` statements containing subqueries. [#8970](https://github.com/cockroachdb/cockroach/pull/8970)
+- The [`TRUNCATE`](truncate.html) statement now implements the `CASCADE` modifier. [#9240](https://github.com/cockroachdb/cockroach/pull/9240)
+- [`TIMESTAMP`](timestamp.html) values that include a time zone can now omit the minutes field of the time zone offset, for compatibility with PostgreSQL. [#8666](https://github.com/cockroachdb/cockroach/pull/8666)
+- [`INTERVAL`](interval.html) values can now be input in a colon-delimited format (H:M or H:M_S), for compatibility with PostgreSQL. [#8603](https://github.com/cockroachdb/cockroach/pull/8603)
+- The `INT8` type is now supported as an alias for [`INTEGER`](int.html). [#8858](https://github.com/cockroachdb/cockroach/pull/8858)
+- [`INTERVAL`](interval.html), [`TIMESTAMP`](timestamp.html), [`TIMESTAMPTZ`](timestamp.html), and [`DATE`](date.html) values can now be casted to/from more types. [#9731](https://github.com/cockroachdb/cockroach/pull/9731)
+- The window functions `row_number()`, `rank()`, `dense_rank()`, `percent_rank()`, `cume_dist()`, `ntile()`, `lead()`, `lag()`, `first_value()`, `last_value()`, `nth_value()` are now supported. [#8928](https://github.com/cockroachdb/cockroach/pull/8928), [#9321](https://github.com/cockroachdb/cockroach/pull/9321), [#9335](https://github.com/cockroachdb/cockroach/pull/9335)
```

Comment:
I've been inconsistent in capitalization for function names. I used lowercase here and uppercase below for EXP(), ROUND(), etc. Looks like all the function names below should be lowerecased to match our functions & operators page.